tests/build.test.js

```diff
@@ -0,0 +1,8 @@
+const { execSync } = require('child_process');
+
+test('build runs without an error', () =>
+  expect(() =>
+    execSync('node ../bin build', {
+      cwd: 'tests',
```

Comment:
I think I liked the calls to `execSync` with `node bin` more. Let's maybe change this to stay consistent with the other tests :)

Reply:
`build` looks for a `src` folder. Therefore the working directory has to be the `test` directory. :confused:

Reply:
Ahhh... right... we don't have support for targets in the build script. You're right!